src/ray/core_worker/task_event_buffer.cc

```diff
@@ -53,6 +89,17 @@ Status TaskEventBufferImpl::Start(bool auto_flush) {
     sigaddset(&mask, SIGINT);
     sigaddset(&mask, SIGTERM);
     pthread_sigmask(SIG_BLOCK, &mask, NULL);
+
+    // Decrease the thread priority to allow worker threads to run.
+    int new_nice = std::min(RayConfig::instance().worker_niceness() + 5, 19);
+    new_nice = nice(new_nice);
+    if (new_nice == -1) {
+      RAY_LOG(WARNING) << "Failed to set lower priority for task event buffer io thread: "
```

Comment:
nit: add `strerror`, example: https://github.com/ray-project/ray/pull/30150/files#diff-ee73f8b8b8cf33a817e1d8069d847afba82da19fc7ca011f2b96d2f7a2cac29dR58
nit: include `new_nice` in error message to help to make log more useful